docs/core/whats-new/dotnet-10/overview.md

```diff
@@ -97,6 +97,6 @@ For more information, see [What's new in Windows Forms for .NET 10](/dotnet/desk
 
 ## See also
 
-- [.NET 10 Preview 1 container image updates](https://github.com/dotnet/core/blob/dotnet10p1/release-notes/10.0/preview/preview1/containers.md)
-- [F# updates in .NET 10 Preview 1](https://github.com/dotnet/core/blob/dotnet10p1/release-notes/10.0/preview/preview1/fsharp.md)
-- [Visual Basic updates in .NET 10 Preview 1](https://github.com/dotnet/core/blob/dotnet10p1/release-notes/10.0/preview/preview1/visualbasic.md)
+- [.NET 10 Preview 1 container image updates](https://github.com/dotnet/core/blob/main/release-notes/10.0/preview/preview1/containers.md)
```

Comment:
It looks like the branch name became dotnet-10p1-sdk-release-notes. But still containers.md isn't there. I can see it in p2 branch though: https://github.com/dotnet/core/blob/dotnet10p2/release-notes/10.0/preview/preview1/containers.md (but still under preview1 directory). Maybe the file is missing on the right branch?